build.gradle

```diff
@@ -219,10 +210,11 @@ release {
 
 task jacocoAggregateReport(type: org.gradle.testing.jacoco.tasks.JacocoReport) {
     dependsOn = subprojects.test
-    additionalSourceDirs = files(subprojects.sourceSets.main.allSource.srcDirs)
-    sourceDirectories = files(subprojects.sourceSets.main.allSource.srcDirs)
-    classDirectories = files(subprojects.sourceSets.main.output)
-    executionData = files(subprojects.jacocoTestReport.executionData)
+    getAdditionalSourceDirs().setFrom(files(subprojects.sourceSets.main.allSource.srcDirs))
```

Comment:
these properties have been changed to getters and setters
https://docs.gradle.org/6.0-rc-3/userguide/upgrading_version_5.html#other_deprecated_behaviors_and_apis